build.gradle

```diff
@@ -40,13 +40,15 @@ test {
 }
 
 dependencies {
-	compile('org.springframework.boot:spring-boot-starter-web')
-	compile('org.springframework.boot:spring-boot-starter-data-jpa')
-	compile('org.springframework.boot:spring-boot-devtools')
-    compile("com.h2database:h2")
-
-	// Test dependent
-	testCompile('org.springframework.boot:spring-boot-starter-test')
+    compile('org.springframework.boot:spring-boot-starter-web')
+    compile('org.springframework.boot:spring-boot-starter-data-jpa')
+    compile('org.springframework.boot:spring-boot-devtools')
+//    compile("org.springframework.boot:spring-boot-starter-security")
```

Comment:
why is this commented tho? if this isn't needed just yet, better erase it

Reply:
okeey, will remove it